src/wallet/wallet.h

```diff
@@ -52,6 +54,9 @@ static const unsigned int DEFAULT_TX_CONFIRM_TARGET = 2;
 static const CAmount nHighTransactionMaxFeeWarning = 100 * nHighTransactionFeeWarning;
 //! Largest (in bytes) free transaction we're willing to create
 static const unsigned int MAX_FREE_TRANSACTION_CREATE_SIZE = 1000;
+//! Size of witness cache
+//  Should be large enough that we can expect to never reorg beyond our cache.
```

Comment:
... that we expect not to reorg beyond our cache unless there is some exceptional network disruption.